dlhub_sdk/models/servables/python.py

```diff
@@ -163,28 +176,102 @@ def create_model(cls, module, method, autobatch=False, function_kwargs=None):
                 Calls :code:`map(f, list)`
             function_kwargs (dict): Names and values of any other argument of the function to set
                 the values must be JSON serializable.
+            f (object): A function pointer
+            auto_inspect (boolean): Whether or not to attempt to automatically extract inputs from the function
+        Raises:
+            TypeError: If there is no valid way to process the given arguments
         """
+        if f is not None:
```

Comment:
please add some inline comments here explaining the logic for the user. they can be short, since the docstring for this function is quite good!